backend/server/src/test/java/ai/verta/modeldb/DatasetTest.java

```diff
@@ -612,12 +610,12 @@ public void k_getDatasetByNameWithWorkspace() {
     CreateDataset createDatasetRequest = getDatasetRequest("Dataset-" + new Date().getTime());
     var workspaceNameUser2 = testUser2.getVertaInfo().getUsername();
     if (testConfig.isPermissionV2Enabled() && !isRunningIsolated()) {
-      var workspaceStub = WorkspaceServiceV2Grpc.newBlockingStub(authServiceChannelServiceUser);
+      /*var workspaceStub = WorkspaceServiceV2Grpc.newBlockingStub(authServiceChannelServiceUser);
```

Comment:
Can we leave a comment here about why the code is commented out? I prefer to either remove it entirely, or else include a comment about when it should be uncommented.

Reply:
this should be not there. I will remove it.